playwright/integrations/unauthenticated-chat-with-ooh.spec.ts

```diff
@@ -0,0 +1,40 @@
+import fetchOmnichannelConfig from '../utils/fetchOmnichannelConfig';
+import fetchTestPageUrl from '../utils/fetchTestPageUrl';
+import { test, expect } from '@playwright/test';
+
+const testPage = fetchTestPageUrl();
+const omnichannelConfig = fetchOmnichannelConfig('UnauthenticatedChatOutsideOfOperatingHours');
+
+test.describe('UnauthenticatedChat @UnauthenticatedChatOutsideOfOperatingHours', () => {
```

Comment:
Please rename the file as well to `unauthenticated-chat-outside-of-operating-hours.ts`